netboot-oci-artifacts.md

```diff
@@ -1,44 +1,66 @@
-The OCI image manifest MUST NOT contain any associated configuration files required for booting.
+The main storage layout is described in [OCI Artifacts](https://github.com/opencontainers/image-spec/blob/main/artifacts-guidance.md)
+specification using an [OCI image manifest](https://github.com/opencontainers/image-spec/blob/master/manifest.md)
+and stored in an OCI registry.
 
+## Notational Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119) (Bradner, S., "Key words for use in RFCs to Indicate Requirement Levels", BCP 14, RFC 2119, March 1997).
+
+The key words "unspecified", "undefined", and "implementation-defined" are to be interpreted as described in the [rationale for the C99 standard][c99-unspecified].
+
+An implementation is not compliant if it fails to satisfy one or more of the MUST, MUST NOT, REQUIRED, SHALL, or SHALL NOT requirements for the protocols it implements.  An implementation is compliant if it satisfies all the MUST, MUST NOT, REQUIRED, SHALL, and SHALL NOT requirements for the protocols it implements.
+
+All required files MUST be packaged and stored in the registry together in order to either boot into container, or boot into installer which is responsible for deploying bootable image. The OCI image manifest MUST NOT contain any associated configuration files required for booting.
 
 ## Image config object
 
-The configuration object MUST have the MIME type `application/vnd.oci.empty.v1+json`, and MUST be
-an empty JSON object ('{}').
+The configuration object MUST have the MIME type `application/vnd.oci.empty.v1+json`, and MUST be an empty JSON object ('{}').
 See the [Guidance for an empty descriptor](https://github.com/opencontainers/image-spec/blob/main/manifest.md#guidance-for-an-empty-descriptor) 
 
-Because the `config.mediaType` is set to the empty value, the `artifactType` MUST be defined and
-currently it has MIME type `application/vnd.unknown.artifact.v1`.
-
+Because the `config.mediaType` is set to the empty value, the `artifactType` MUST be defined and currently it has MIME type `application/vnd.unknown.artifact.v1`.
 
 ## Image layers
 
-Each layer in the manifest MUST map to one Netboot file provided at build time.
+The image layer is defined in the [OCI specs](https://github.com/opencontainers/image-spec) with the following clauses.
+
+Each layer in the manifest MUST map to exactly one Netboot file provided at build time. For example, in order to boot Red Hat compatible x86_64 system via PXE, the following files are subject of storing into layers:
+
+* `shim.efi`
+* `grubx64.efi`
+* `vmlinuz`
+* `initrd.img`
+* `install.img`
+
+Each file MUST be compressed via zstandard compression algorithm. Each layer MUST have the MIME type of `application/x-netboot-file+zstd` and contain the following annotations:
```

Comment:
s/zstandard/standard

Reply:
_Zstandard, or zstd as short version, is a fast lossless compression algorithm, targeting real-time compression scenarios at zlib-level and better compression ratios._

Reply:
Zstandard, or zstd as short version, is a fast lossless compression algorithm, targeting real-time compression scenarios at zlib-level and better compression ratios.

Reply:
but i just googled and you are right

Reply:
ah now I get what was the concern, I can use `zstd` acronym sure